README.md

```diff
@@ -94,6 +102,7 @@ public NexmoClient.Builder customNexmoBuilder() {
 ```
 > Note that you must include your credentials as shown in this example. This builder completely replaces the automatically configured one.
 
+<a name="customization_version"></a>
```

Comment:
HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.

Reply:
Modified